packages/sui-analytics/src/events/index.js

```diff
@@ -0,0 +1,3 @@
+export {default as identify} from './identify'
+export {default as track} from './track'
+export {default as screen} from './screen'
```

Comment:
Not sure if screen is the most appropriated name for page views... 🤔

Reply:
me either, one pro of calling it screen is that we will have a common language with data and apps

Reply:
Use the same language than Data and Apps I think that will be the best.

Reply:
You're right, in Motor Data&Insights are already using "screen" concept also for page views 👍